cmd/celestia/bridge/bridge.go

```diff
@@ -0,0 +1,54 @@
+package bridge
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	cmdnode "github.com/celestiaorg/celestia-node/cmd"
+	"github.com/celestiaorg/celestia-node/nodebuilder/core"
+	"github.com/celestiaorg/celestia-node/nodebuilder/gateway"
+	"github.com/celestiaorg/celestia-node/nodebuilder/node"
+	"github.com/celestiaorg/celestia-node/nodebuilder/p2p"
+	"github.com/celestiaorg/celestia-node/nodebuilder/rpc"
+	"github.com/celestiaorg/celestia-node/nodebuilder/state"
+)
+
+// NOTE: We should always ensure that the added Flags below are parsed somewhere, like in the
+// PersistentPreRun func on parent command.
+
+func WithSubcommands() func(*cobra.Command, []*pflag.FlagSet) {
+	return func(c *cobra.Command, flags []*pflag.FlagSet) {
+		c.AddCommand(
+			cmdnode.Init(flags...),
+			cmdnode.Start(cmdnode.WithFlagSet(flags)),
+			cmdnode.AuthCmd(flags...),
+			cmdnode.ResetStore(flags...),
+			cmdnode.RemoveConfigCmd(flags...),
+			cmdnode.UpdateConfigCmd(flags...),
+		)
+	}
+}
```

Comment:
this is duplicated across the three node type packages now, i think for the purpose of this PR we can leave it, but we should break apart some of this into some builder pattern that just takes the node specific type of customization/config (note to self)

Reply:
Huge +1. We should move the WithSubcommands to the main pkg where its used. All node types will always have the same set of subcmds, and if that happens to be not, we will separate. I think we should move to the main in the context of this PR though

Reply:
Moved 👍🏽

Reply:
ok agree, @tuxcanfly can we do this quickly?